tests/integration_tests_control_files/euler_entropy_conservation/euler_ismail_roe_entropy_check.prm

```diff
@@ -0,0 +1,53 @@
+# Listing of Parameters
+# ---------------------
+# Number of dimensions
+
+set dimension = 3
+set run_type = integration_test
+set test_type = euler_ismail_roe_entropy_check
+set pde_type = euler
+
+# DG formulation
+set use_weak_form = false
+set use_split_form = true
+set use_collocated_nodes = true
+
+set flux_reconstruction = cDG
+set use_inverse_mass_on_the_fly = true
+# set use_collocated_nodes = true
+
+# Note: this was added to turn off check_same_coords() -- has no other function when dim!=1
+set use_periodic_bc = true
+
+# numerical fluxes
+set conv_num_flux = two_point_flux
+set two_point_num_flux_type = IR
+
+# ODE solver
+subsection ODE solver
+  set ode_output = quiet
+  set ode_solver_type = runge_kutta
+  set output_solution_every_x_steps = 10
+  set runge_kutta_method = rk4_ex
+end
+
+# freestream Mach number
+subsection euler
+  set mach_infinity = 0.1
+end
+
+subsection flow_solver
+  set flow_case_type = taylor_green_vortex
+  set poly_degree = 3 
+  set final_time = 0.1
+  set courant_friedrich_lewy_number = 0.005
+  set unsteady_data_table_filename = tgv_kinetic_energy_vs_time_table
+  subsection grid
+    set grid_left_bound = 0.0
+    set grid_right_bound = 6.28318530717958623200
+    set number_of_grid_elements_per_dimension = 4
+  end
+  subsection taylor_green_vortex
+    set do_calculate_numerical_entropy = true
```

Comment:
can you also test it for the `set density_initial_condition_type = isothermal` ? Its `uniform` by default but the isothermal initialization is the most popular

Reply:
Thanks for the suggestion, I didn't know about the different initializations! The isothermal condition also passes for the short simulation time in this test.

Reply:
I will commit the .prm file with isothermal initialization, since it is the more popular one.